HW03-covid.md

```diff
@@ -0,0 +1,92 @@
+HW03 covid
+================
+Bei Wang
+July 19, 2020
+
+This dataset if from NYT about COVID-19 cases in the US from Feb2020 to July2020.
+
+``` r
+#packages used 
+
+library("ggplot2")
+library("dplyr")
+```
+
+    ## 
+    ## Attaching package: 'dplyr'
+
+    ## The following objects are masked from 'package:stats':
+    ## 
+    ##     filter, lag
+
+    ## The following objects are masked from 'package:base':
+    ## 
+    ##     intersect, setdiff, setequal, union
+
+``` r
+library("magrittr")
+```
+
+Covid-19 in the US
+------------------
+
+``` r
+#import datasets downloaded from NYT
+national_data<- read.csv("https://raw.githubusercontent.com/nytimes/covid-19-data/master/us.csv", header=TRUE)
+#transform "date" to date format
+national_data$date<- as.Date(national_data$date)
+
+#cases plot
+national_caseplot<- national_data %>% 
+  ggplot()+
+  geom_col(aes(x=date, y=cases), color="#69b3a2", fill="grey", alpha=0.7)+
+  geom_col(aes(x=date, y=deaths), color="red", fill="red", alpha=0.7)+
+  theme_minimal()+
```

Comment:
The use of geom_col() is very clever. In such a way the proportions of deaths among total cases are very clear. One thing I am not very sure is: what do the green and grey areas for cases mean respectively?

Reply:
> The use of geom_col() is very clever. In such a way the proportions of deaths among total cases are very clear. One thing I am not very sure is: what do the green and grey areas for cases mean respectively?

I guess the purpose of using color="#69b3a2" is to show the border of each column. Otherwise the whole area will be in grey. Is that right?

Reply:
Yes, Mingyu is right about this.